src/components/InventoryTable/EntityTableToolbar.js

```diff
@@ -256,12 +255,6 @@ const EntityTableToolbar = ({
       ...activeFilters,
       ...customFilters,
     },
-    onSuccess: () => {
```

Comment:
This is moved into the hook itself to consolidate really everything related to export in the toolbar.